packages/devtools_app/lib/src/screens/inspector_v2/widget_details.dart

```diff
@@ -0,0 +1,94 @@
+// Copyright 2024 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_app_shared/ui.dart';
+import 'package:devtools_app_shared/utils.dart';
+import 'package:flutter/widgets.dart';
+
+import '../../shared/console/eval/inspector_tree_v2.dart';
+import '../../shared/diagnostics/diagnostics_node.dart';
+import 'inspector_controller.dart';
+import 'inspector_screen.dart';
+import 'layout_explorer/box/box.dart';
+import 'layout_explorer/flex/flex.dart';
+import 'widget_properties/properties_view.dart';
+
+/// Panes showing details pertaining to the selected widget.
+///
+/// Includes both the [FlexLayoutExplorerWidget] or [BoxLayoutExplorerWidget]
+/// and the [PropertiesView].
+class WidgetDetails extends StatefulWidget {
+  const WidgetDetails({super.key, required this.controller});
+
+  final InspectorController controller;
+
+  @override
+  State<WidgetDetails> createState() => _WidgetDetailsState();
+}
+
+class _WidgetDetailsState extends State<WidgetDetails> with AutoDisposeMixin {
+  InspectorController get controller => widget.controller;
+
+  RemoteDiagnosticsNode? get selected =>
+      controller.selectedNode.value?.diagnostic;
+
+  RemoteDiagnosticsNode? previousSelection;
+
+  Widget rootWidget(RemoteDiagnosticsNode? node) {
```

Comment:
avoid creating methods that return widgets. The build method below is pretty short, so I think it is reasonable to include all this widget code directly in the build method

Reply:
Oops thank you! This was a remnant of this file being a rename of the layout_explorer file (which for some reason git did not track)